examples/002_visualizations.py

```diff
@@ -0,0 +1,95 @@
+"""Example 2: Visualizations
+
+This example demonstrates the use of the flow and layout visualizations in FLORIS.
```

Comment:
Here it would be helpful to explain a little more what the example looks at. After the above sentence, something like: "First, an example wind farm layout is plotted, with the turbine names and the directions and distances between turbines shown. Next, the horizontal flow field at hub height is plotted for a single wind condition." I think this level of detail (a few sentences) would be good for all the examples.

Reply:
Done!  I'll make a pass through

Reply:
@ejsimley I went through the examples and tried to beef up the docstrings at the top of each